company_website/templates/main_page_partials/main_screen.haml

```diff
@@ -1,8 +1,13 @@
 {% load static %}
+{% load variable_tag %}
 
 .container-fluid.px-0.main-screen{:id => "home"}
     #content.jumbo.d-flex
+        {% if request.user_agent.browser.family|is_webp_supported %}
```

Comment:
Why here you haven't used same mechanism as on other pages? We shouldn't duplicate code, please fix this :)